sdks/java/io/jms/src/main/java/org/apache/beam/sdk/io/jms/JmsRecord.java

```diff
@@ -31,8 +30,8 @@
   private final String jmsMessageID;
   private final long jmsTimestamp;
   private final String jmsCorrelationID;
-  private final Destination jmsReplyTo;
-  private final Destination jmsDestination;
+  private final String jmsReplyTo;
+  private final String jmsDestination;
```

Comment:
Per the [documentation](http://docs.oracle.com/javaee/6/api/javax/jms/Destination.html):

> JMS provider implementations of administered objects should implement the `javax.naming.Referenceable` and `java.io.Serializable` interfaces so that they can be stored in all JNDI naming contexts.

Up to you, but it might be okay to leave the type as `Destination` and instead suppress the Findbugs message, documenting the reason. E.g., https://github.com/apache/incubator-beam/blob/5864a38ba595cdf90d2d5559e4312ef6144f60c7/sdks/java/core/src/main/java/org/apache/beam/sdk/util/ExposedByteArrayInputStream.java#L36